frontend/src/components/BlogCard.tsx

```diff
@@ -46,7 +46,7 @@ const BlogCard = ({ author, title, content, publishedDate, id, fullWidth, tagsOn
       <div className="order-3 md:order-none md:flex col-span-full pb-8 px-2 border-b md:border-none border-gray-100">
         <div className="flex">
           {tagsOnPost?.slice(0, 2).map((tagWrapper) => {
-            return <Pill id={tagWrapper.tag.id} tagName={tagWrapper.tag.tagName} />;
+            return <Pill key={tagWrapper.tag.id} tagName={tagWrapper.tag.tagName} />;
```

Comment:
It's great that you've added key. but please keep the id property. 
otherwise the build fails
![Screenshot 2024-06-13 at 10 34 07 AM](https://github.com/aadeshkulkarni/medium-app/assets/67855887/c7820602-3750-4a78-98c2-9af50c83ae33)